src/app/pages/ledger/connect-ledger-error/connect-ledger-error.layout.tsx

```diff
@@ -0,0 +1,75 @@
+import { FiCircle } from 'react-icons/fi';
+import { Box, color, Flex, Stack } from '@stacks/ui';
+
+import { BaseDrawer } from '@app/components/drawer';
+import { Caption } from '@app/components/typography';
+import ConnectLedgerError from '@assets/images/onboarding/ledger/connect-ledger-error.png';
+import { ErrorLabel } from '@app/components/error-label';
+import { PrimaryButton } from '@app/components/primary-button';
+import { Link } from '@app/components/link';
+
+import { ConnectLedgerTitle } from '../components/connect-ledger-title';
+
+interface PossibleReasonUnableToConnectProps {
+  text: string;
+}
+function PossibleReasonUnableToConnect(props: PossibleReasonUnableToConnectProps) {
+  const { text } = props;
+
+  return (
+    <Flex>
+      <Box mr="tight" mt="3px">
+        <FiCircle fill={color('text-caption')} size="4px" />
+      </Box>
+      <Caption>{text}</Caption>
+    </Flex>
+  );
+}
+
+interface ConnectLedgerErrorLayoutProps {
+  onCancelConnectLedger(): void;
+  onTryAgain(): void;
+}
+export function ConnectLedgerErrorLayout(props: ConnectLedgerErrorLayoutProps) {
+  const { onCancelConnectLedger, onTryAgain } = props;
+
+  return (
+    <BaseDrawer title={<Box />} isShowing onClose={onCancelConnectLedger}>
+      <Stack alignItems="center" pb="loose" px="loose" spacing="loose" textAlign="center">
+        <Box mt="tight" width="247px">
+          <img src={ConnectLedgerError} />
+        </Box>
+        <ConnectLedgerTitle />
+        <ErrorLabel fontSize="14px" lineHeight={1.4}>
```

Comment:
I think I've already asked this question, I am sorry if I am asking it again. Doesn't the system have a standard for the error message font size? and if we need to change it here do we have a variable for 14px?

Reply:
The `ErrorLabel` is a component in the wallet not the UI library and it has a default font size of `1rem`.  I don't know of a variable but can use `fontSize={1}` here.

Reply:
If `{1}`= `1rem`, then no. The default setting for 1rem in the browser is 16px. Again, please ignore this comment, I don't have context of the app nor the design system's variables.

Reply:
`{1}` in our case is 14px. See here in the UI library, I think this is what determines those numbers: https://github.com/hirosystems/ui/blob/c9efa75863b29b5f30302b38009dded893cf9b0c/packages/ui-core/src/css/index.ts#L39